components/tag-list/demo/tag-list.html

```diff
@@ -42,31 +42,47 @@ <h2>Basic Tag List</h2>
 			</script>
 			<d2l-demo-snippet fullscreen>
 				<d2l-tag-list description="A bunch of example tags">
-					<d2l-tag-list-item text="Example Tag"></d2l-tag-list-item>
```

Comment:
I changed the text in these items to make unit testing the announcements easier, given the role descriptions that are announced (i.e. repetitive "Tag" is somewhat confusing).